src/config.rs

```diff
@@ -2467,19 +2467,15 @@ pub struct BackupStreamConfig {
     #[online_config(skip)]
     pub num_threads: usize,
     #[online_config(skip)]
-    pub io_threads: usize,
-    #[online_config(skip)]
     pub enable: bool,
     #[online_config(skip)]
     pub temp_path: String,
     #[online_config(skip)]
-    pub temp_file_size_limit_per_task: ReadableSize,
+    pub file_size_limit: ReadableSize,
```

Comment:
Does this will impact the existing cluster that has dependency with the old parameter?

Reply:
This config field is added only for log-backup( the feature about `PiTR`) that will be planned to release in 6.2.